packages/web/components/your-balance/your-balance.tsx

```diff
@@ -0,0 +1,170 @@
+import { observer } from "mobx-react-lite";
+import Image from "next/image";
+import Link from "next/link";
+import { ReactElement } from "react";
+
+import { RightArrowIcon } from "~/components/assets/right-arrow-icon";
+import { useStore } from "~/stores";
+
+import explorePools from "../../public/images/explore-pools.svg";
+import stakingAPRFull from "../../public/images/staking-apr-full.svg";
```

Comment:
You don't need to use import in that way, just use paths inside "src" prop:
https://github.com/nabla-studio/osmosis-frontend/blob/fabryscript/feat-your_balance_section/packages/web/components/overview/pools.tsx#L89